lib/modules/platform/azure/pr-cache.spec.ts

```diff
@@ -0,0 +1,121 @@
+import type { GitPullRequest } from 'azure-devops-node-api/interfaces/GitInterfaces';
+import type { Mocked } from 'vitest';
+import { mockDeep } from 'vitest-mock-extended';
+import { reset as memCacheReset } from '../../../util/cache/memory';
+import {
+  getCache,
+  resetCache as repoCacheReset,
+} from '../../../util/cache/repository';
+import { AzurePrCache } from './pr-cache';
+import { getRenovatePRFormat } from './util';
+
+vi.mock('./azure-got-wrapper', () => mockDeep());
+
+const pr1: GitPullRequest = {
+  pullRequestId: 1,
+  title: 'title',
+  status: 1,
+  sourceRefName: 'branch',
+  targetRefName: 'master',
+  reviewers: [],
+  labels: [],
+  description: 'a merge request',
+};
+
+const pr2: GitPullRequest = {
+  pullRequestId: 2,
+  title: 'title',
+  status: 1,
+  sourceRefName: 'branch',
+  targetRefName: 'master',
+  labels: [],
+  description: 'a merge request',
+};
+
+describe('modules/platform/azure/pr-cache', () => {
+  let azureApi: Mocked<typeof import('./azure-got-wrapper')>;
+  let cache = getCache();
+
+  beforeAll(() => {
+    vi.useFakeTimers();
+  });
+
+  beforeEach(async () => {
+    azureApi = await vi.importMock('./azure-got-wrapper');
```

Comment:
do we really need to import it again and again? (perf)

I don't see any reset modules call, so i assume we can statically import it for better test perf.